lib/awful/hotkeys_popup/keys/qutebrowser.lua

```diff
@@ -0,0 +1,187 @@
+---------------------------------------------------------------------------
+--- Qutebrowser hotkeys for awful.hotkeys_widget
+--
+-- @author Simon Désaulniers sim.desaulniers@gmail.com
+-- @copyright 2017 Simon Désaulniers
+-- @module awful.hotkeys_popup.keys.qutebrowser
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local qutebrowser_rule_any = {class={"qutebrowser", "qutebrowser"}}
```

Comment:
Cool!
Use a normal `rule` then instead of `rule_any`, and just the class (and not a table/list).